### PR TITLE
Added new page on Q# in research.

### PR DESCRIPTION
**Pull request by cgranade**
This PR adds a new page on using Q# in research.

---

In draft for early feedback.

Still TODO before bringing out of draft:
- [ ] collect list of existing papers that use Q#.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/resources/research.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/59bbfea6dbeef257da315b5b597aab9f6897f0d3/articles/resources/research.md) | 83 | 57 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a8864ea-b7de-4626-8ada-bb7ad568c8e7) | [:white_check_mark:](# "Successfully processed") |
| [articles/resources/toc.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/59bbfea6dbeef257da315b5b597aab9f6897f0d3/articles/resources/toc.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ae790e07-e31a-4709-ad6e-f734bdaf39be) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/resources/research.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+The Quantum Development Kit provides several tools to help include Q# code into a reproducible research workflow:
+
+- [**qsharp.sty**](https://github.com/msr-quarc/qsharp.sty): Syntax highlighting support for LaTeX.
+- [**quantum-research-template**](https://github.com/cgranade/quantum-research-template): Example project using Q#, LaTeX, and Python together, pre-configured for use in [Visual Studio Codespaces](https://visualstudio.microsoft.com/services/visual-studio-codespaces/).
+
+## Papers using Q\#
+
+- _Implementing Grover Oracles for Quantum Key Search on AES and LowMC_, Jaques et al. (2020). DOI [10/fmcx](https://doi.org/fcmx).
+- _Property-based Testing of Quantum Programs in Q#_, Honarvar, Mousavi, and Nagarajan (2020). DOI [10/fcmz](https://doi.org/fcmz).
+Google's arithmetic papers
```
Comment (geduardo) [inline articles/resources/research.md:40]:
```suggestion
Google's arithmetic papers
```
This seems out of place (?), maybe it was a todo.

File: articles/resources/research.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ms.date: 8/5/2020
+ms.topic: article
+uid: microsoft.quantum.research
+no-loc: ['Q#', '$$v']
+---
+
+# Q# in Research
+
+Q# and the Quantum Development Kit are important tools for empowering quantum computing research.
+On this page, we list a few resources to help you get the most out of Q# in your research.
```
Comment (geduardo) [inline articles/resources/research.md:15]: I'd tone down the introductory paragraph and make it a be more descriptive. This page is targeted to researchers and they tend to like a more aseptic language.

**Comment by KittyYeungQ** [comment]
Closing this for now before repo migration is done. Then we can decide if we will implement it.

**Pull request closed without merging**